modules/virt-importing-rhel-image-boot-source-web.adoc

```diff
@@ -4,35 +4,30 @@
 
 :_content-type: PROCEDURE
 [id="virt-importing-rhel-image-boot-source-web_{context}"]
-= Importing a Red Hat Enterprise Linux image as a boot source
+= Importing a {op-system-base} image as a boot source
 
-You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by specifying the URL address for the image.
+You can import a {op-system-base-full} image as a boot source by specifying a URL for the image.
 
 .Prerequisites
-* You must have access to the web server with the operating system image. For example: Red Hat Enterprise Linux web page with images.
 
-.Procedure
+* You must have access to a web page with the operating system image. For example: Download Red Hat Enterprise Linux web page with images.
 
-. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+.Procedure
 
-. Click the *Templates* tab.
+. In the {VirtProductName} console, click *Virtualization* -> *Templates* from the side menu.
```

Comment:
```suggestion
. In the {product-title} console, click *Virtualization* -> *Templates* from the side menu.
```

Reply:
Totally agree. I hated seeing the Virt console everywhere. Will change this everywhere.